org.gnome.Evolution.json

```diff
@@ -0,0 +1,375 @@
+                    "-DGOBJECT_INTROSPECTION:BOOL=OFF"
+                ],
+                "sources": [
+                    {
+                        "type": "archive",
+                        "url": "https://github.com/libical/libical/releases/download/v3.0.3/libical-3.0.3.tar.gz",
+                        "sha256": "5b91eb8ad2d2dcada39d2f81d5e3ac15895823611dc7df91df39a35586f39241"
+                    }
+                ]
+            },
+     
+            {
+                "name": "liboauth",
+                "config-opts": [ "--enable-nss" ],
+                "cleanup": [ "/share/man" ],
+                "sources": [
+                    {
+                        "type": "archive",
+                        "url": "https://downloads.sourceforge.net/project/liboauth/liboauth-1.0.3.tar.gz",
+                        "sha256": "0df60157b052f0e774ade8a8bac59d6e8d4b464058cc55f9208d72e41156811f"
+                    }
+                ]
+            },
+     
+            {
+                "name": "gnome-online-accounts",
+                "config-opts": 
+		[ 
+		"--disable-Werror",
+		"--disable-telepathy",
+		"--disable-documentation",
+		"--disable-backend" 
+		],
+                "cleanup": [ "/bin", "/share/GConf" ],
+                "sources": [
+                    {
+                        "type": "archive",
+                        "url": "https://download.gnome.org/sources/gnome-online-accounts/3.28/gnome-online-accounts-3.28.0.tar.xz",
+                        "sha256": "87bc4ef307604f1ce4f09f6e5c9996ef8d37ca5e0a3bf76f6b27d71844adb40c"
+                    }
+                ]
+            },
+     
+            {
+                "name": "libgdata",
+                "config-opts": [
+                    "--disable-always-build-tests",
+                    "--disable-goa",
+                    "--disable-gtk-doc",
+                    "--disable-introspection",
+                    "--disable-installed-tests",
+                    "--disable-vala"
+                ],
+                "cleanup": [
+                    "/share/gtk-doc"
+                ],
+                "sources": [
+                    {
+                        "type": "archive",
+                        "url": "https://download.gnome.org/sources/libgdata/0.17/libgdata-0.17.9.tar.xz",
+                        "sha256": "85c4f7674c0098ffaf060ae01b6b832cb277b3673d54ace3bdedaad6b127453a"
+                    }
+                ]
+            },
+     
+            {
+                "name": "geocode-glib",
+                "config-opts": [
+                    "--disable-introspection",
+                    "--disable-gtk-doc"
+                ],
+                "cleanup": [
+                    "/share/gtk-doc"
+                ],
+                "sources": [
+                    {
+                        "type": "archive",
+                        "url": "https://download.gnome.org/sources/geocode-glib/3.24/geocode-glib-3.24.0.tar.xz",
```

Comment:
Does evo work with a more recent geocode-glib? https://download.gnome.org/sources/geocode-glib/3.26/

Reply:
It seems to work just fine with the newer version, I'm going to update the manifest.